website/src/index.tsx

```diff
@@ -1,14 +1,29 @@
 import React from "react";
-import ReactDOM from "react-dom";
+// import ReactDOM from "react-dom";
+import ReactDOM from "react-dom/client";
 import "./index.css";
 import App from "./App";
 import reportWebVitals from "./reportWebVitals";
+// ================================================
+// deperacted rendering for the Radis app
+// ================================================
 
-ReactDOM.render(
+// ReactDOM.render(
+//   <React.StrictMode>
+//     <App />
+//   </React.StrictMode>,
+//   document.getElementById("root")
+// );
+// ================================================
+// deperacted rendering for the Radis app
+// ================================================
```

Comment:
We can just remove this entirely instead of leaving it in the main branch. If we need to refer to it in the future, it's in version control.